com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -3392,7 +3337,11 @@ internal enum TransformExportType { Local, Global, Reset };
                     // The Maya axis system has Y up, Z forward, X to the left (right handed system with odd parity).
                     // We need to export right-handed for Maya because ConvertScene can't switch handedness:
                     // https://forums.autodesk.com/t5/fbx-forum/get-confused-with-fbxaxissystem-convertscene/td-p/4265472
-                    fbxSettings.SetAxisSystem (FbxAxisSystem.MayaYUp);
+                    var unityAxisSystem = new FbxAxisSystem(
+                        FbxAxisSystem.EUpVector.eYAxis,
+                        FbxAxisSystem.EFrontVector.eParityOdd,
+                        FbxAxisSystem.ECoordSystem.eLeftHanded);
+                    fbxSettings.SetAxisSystem (unityAxisSystem);
```

Comment:
Let's use the DirectX constant (which is this convention). Then when we switch to what Jeremy commented we should use, we get no confusion what "Unity" convention means for FBX.